horizon/state.py

```diff
@@ -183,31 +177,25 @@ def _build_state_payload(cls, state: Optional[PersistentState] = None) -> dict:
             },
         }
 
-    async def reporter_user_data_handler(
-        self, report: DataUpdateReport
-    ) -> dict[str, Any]:
+    async def reporter_user_data_handler(self) -> dict[str, Any]:
         return {
             "pdp_instance_id": self.get().pdp_instance_id,
         }
 
     @classmethod
-    async def build_state_payload(cls, state: Optional[PersistentState] = None) -> dict:
+    async def build_state_payload(cls) -> dict:
         payload = cls._build_state_payload()
-        payload["state"].update(
-            await asyncio.get_event_loop().run_in_executor(None, cls.get_runtime_state)
-        )
+        payload["state"].update(await asyncio.get_event_loop().run_in_executor(None, cls.get_runtime_state))
         return payload
 
     @classmethod
-    def build_state_payload_sync(cls, state: Optional[PersistentState] = None) -> dict:
+    def build_state_payload_sync(cls) -> dict:
```

Comment:
who used it ? make sure the usages are changed also

Reply:
No one is passing that